nucypher/cli/painting.py

```diff
@@ -854,41 +855,47 @@ def echo_solidity_version(ctx, param, value):
 
 def paint_worklock_status(emitter, registry: BaseContractRegistry):
     from maya import MayaDT
-    worklock_agent = ContractAgency.get_agent(WorkLockAgent, registry=registry)
+    worklock_agent = ContractAgency.get_agent(WorkLockAgent, registry=registry)  # type: WorkLockAgent
     blockchain = worklock_agent.blockchain
 
-    # Agency
-    token_agent = ContractAgency.get_agent(NucypherTokenAgent, registry=registry)
-
     # Time
-    start = MayaDT(worklock_agent.contract.functions.startBidDate().call())
-    end = MayaDT(worklock_agent.contract.functions.endBidDate().call())
-    duration = end - start
-    remaining = end - maya.now()
+    bidding_start = MayaDT(worklock_agent.contract.functions.startBidDate().call())
+    bidding_end = MayaDT(worklock_agent.contract.functions.endBidDate().call())
+    cancellation_end = MayaDT(worklock_agent.contract.functions.endCancellationDate().call())
 
-    # TODO: Include calculated refund and deposit rates
+    bidding_duration = bidding_end - bidding_start
+    cancellation_duration = cancellation_end - bidding_start
+    now = maya.now()
+    bidding_remaining = bidding_end - now if bidding_end > now else timedelta()
```

Comment:
Hmm - what is the intention of `else timedelta()` here?

Reply:
to show zero value instead of negative